module/config.mjs

```diff
@@ -1972,6 +1982,21 @@ DND5E.advancementTypes = {
 
 /* -------------------------------------------- */
 
+let enrichmentLookup;
```

Comment:
Just a minor suggestion to prefix this with an underscore to make it clear that it's being used internally (even though it can't escape the module context). It's a little convention we've used sometimes in config files to do similar things.